### ISSUE TITLE
Getting error message on adding directives and services

### ISSUE THREAD

**Issue by ramsz082** [m7r/grunt-ngdocs#169]
Hi,

In grunt file added below section and it didn't work.

ngdocs : {
		all: ['app/scripts/**/*.js']
	}

So I added folder path for all files individually

ngdocs : {
		all: ['app/scripts/*.js', 'app/scripts/controllers/*.js', 'app/scripts/filters/*.js']
	}

But then when I added directives I got below error.

![image](https://cloud.githubusercontent.com/assets/5945507/9489322/d3f2ca1a-4c12-11e5-8124-8c1820094f1c.png)

Can you please help me on resolving this issue

**Comment by m7r** [comment]
The correct setup should be

```js
ngdocs : {
  all: ['app/scripts/**/*.js']
}
```

Haven't tested grunt on windows. Has the console user write access to this folder?

**Comment by ramsz082** [comment]
I tried the same and got same issue.

![image](https://cloud.githubusercontent.com/assets/5945507/9489639/fa473956-4c14-11e5-8a11-7353b6b44517.png)

**Comment by ramsz082** [comment]
When I add files individually by skipping directives and services. it works fine.
ngdocs : {
all: ['app/scripts/.js', 'app/scripts/controllers/.js', 'app/scripts/filters/*.js']
}

I can see all the html files inside partials/api. So shouldn't be an issue with the folder access. But issue is when I add directives.

**Comment by m7r** [comment]
Is your code online or could you give an example of a not working doc comment?

**Comment by ramsz082** [comment]
Thanks for your reply.

I found the issue. Below is the comment I had in my js file.

/**
 * After a click, disable the element for the threshold period
 *
 * @ngdoc directive
 * @name fredApp.directive:qfxClickSupressThreshold
 * @description
 * # qfxClickSuppressThreshold
 */

When I removed the first line "After a click, disable the element for the threshold period" it worked.

I think ngdocs fails when there is a comment without @.

Am not sure whether its a defect but when I removed that line it worked.
